gulpTasks/filetypes.js

```diff
@@ -154,7 +154,8 @@ function ensureImageIsValid(filetype, image, legacy) {
   // with useless stuff
   if (!legacy) {
     if (!fs.existsSync(path)) {
-      console.warn('Icon ' + path + ' is referenced by file type ' + filetype + ' but cannot be found!');
+      // console.trace();
```

Comment:
It used to be a fatal error, so we would display trace.
The check is faulty so we removed the error for now and made it a warning instead.
What the check was validating actually works.

We will have to update this in a future PR. Or a future commit on this one.